addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/schema/create-or-update-env-type-config.js

```diff
@@ -28,7 +29,7 @@ const schema = {
       type: 'string',
       maxLength: 128,
       minLength: 2,
-      pattern: '^[A-Za-z0-9-_ ]+$',
+      pattern: nameRegex,
     },
     desc: {
```

Comment:
non-block: use `nonHtmlRegex` for desc and estimatedCostInfo pattern. You could also add test coverage.

Reply:
added to CHAM-210 and CHAM-211